components/ActionModals/TableInvoice.vue

```diff
@@ -82,10 +82,17 @@ export default {
       required: true,
     },
   },
-  data: () => ({
-    feeDenom: network.stakingDenom,
-  }),
   computed: {
+    feeDenom: {
+      get() {
+        return this.amounts.length > 1
+          ? network.stakingDenom
+          : this.amounts[0].denom
```

Comment:
this assumes the fee for this denom is specified

Reply:
ah, true